sections/Chapter 6 Sprint 4/retrospective.tex

```diff
@@ -0,0 +1,32 @@
+\section{Retrospective}
+We have received three user stories from the PO, and can use them to plan the next sprint.  We have written 20+ pages in the report. These have been reviewed by team members.  We have accomplished the sprint goals. 
```

Comment:
```suggestion
We have received three user stories from the PO, and can use them to plan the next sprint.  So far, we have written 20+ pages in the report. These have been reviewed by team members.  We have thus accomplished the sprint goals. 
```

Reply:
I don't agree with writing "so far" because this is supposed to be written in past tense, which the text does. Will fix the `thus`, though.

Reply:
```suggestion
We have received three user stories from the PO, and can use them to plan the next sprint.  We have written 20+ pages in the report. These have been reviewed by team members.  We have thus accomplished the sprint goals. 
```